resources/lang/pt-br/users.php

```diff
@@ -144,36 +145,36 @@
             'country_age' => ':age anos de :country.',
         ],
         'page' => [
-            'description' => '<strong>eu!</strong> é uma área pessoal customizável na sua página de perfil.',
-            'edit_big' => 'Edite-me!',
+            'description' => '<strong>eu!</strong> é uma área pessoal personalizável na sua página de perfil.',
+            'edit_big' => 'Editar eu!',
             'placeholder' => 'Digite o conteúdo da página aqui',
             'restriction_info' => "Você precisa ser um <a href='".osu_url('support-the-game')."' target='_blank'>osu!supporter</a> para desbloquear essa função.",
         ],
         'plays_with' => [
             '_' => 'Joga com',
             'keyboard' => 'Teclado',
             'mouse' => 'Mouse',
-            'tablet' => 'Mesa digitalizadora',
-            'touch' => 'Touch Screen',
+            'tablet' => 'Tablet',
+            'touch' => 'Tela sensível ao toque',
         ],
-        'missingtext' => 'Você deve ter feito um erro de digitação! (ou o jogador pode ter sido banido)',
-        'page_description' => 'osu! - Tudo que você sempre quis saber sobre :username!',
+        'missingtext' => 'Você pode ter cometido um erro de digitação! (ou o usuário pode ter sido banido)',
+        'page_description' => 'osu! — Tudo o que você sempre quis saber sobre :username!',
         'rank' => [
             'country' => 'Rank de país para :mode',
             'global' => 'Rank global para :mode',
         ],
         'stats' => [
-            'hit_accuracy' => 'Precisão de Acerto',
-            'level' => 'Level :level',
-            'maximum_combo' => 'Combo Máximo',
-            'play_count' => 'Vezes Jogadas',
-            'ranked_score' => 'Pontuação Rankeada',
-            'replays_watched_by_others' => 'Replays Assistidos por Outros',
-            'score_ranks' => 'Ranks de Pontuação',
-            'total_hits' => 'Hits Totais',
-            'total_score' => 'Pontuação Total',
+            'hit_accuracy' => 'Precisão de acertos',
+            'level' => 'Nível :level',
+            'maximum_combo' => 'Combo máximo',
+            'play_count' => 'Vezes jogadas',
+            'ranked_score' => 'Pontuação rankeada',
```

Comment:
Hm.. I followed the game's translation to avoid conflicts. I'd prefer to translate "ranking" as "classificação", though.